src/fiatconnect/TransferStatusScreen.tsx

```diff
@@ -118,7 +118,7 @@ function SuccessSection({
     ValoraAnalytics.track(FiatExchangeEvents.cico_fc_transfer_success_complete, {
       flow,
       provider,
-      txHash: fiatConnectTransfer.txHash ?? undefined,
+      txHash: fiatConnectTransfer.txHash ?? null,
```

Comment:
```suggestion
      txHash: fiatConnectTransfer.txHash,
```
I think this should work since the state variable is also `string | null`